include/beast/websocket/detail/stream_base.hpp

```diff
@@ -59,14 +59,14 @@ struct stream_base
     std::size_t wr_buf_size_ = 4096;        // write buffer size
     std::size_t rd_buf_size_ = 4096;        // read buffer size
     opcode wr_opcode_ = opcode::text;       // outgoing message type
-    pong_cb pong_cb_;                       // pong callback
+    ping_cb ping_cb_;                       // ping callback
     role_type role_;                        // server or client
     bool failed_;                           // the connection failed
 
     bool wr_close_;                         // sent close frame
     op* wr_block_;                          // op currenly writing
 
-    ping_data* pong_data_;                  // where to put pong payload
+    ping_data* ping_data_;                  // where to put pong payload
```

Comment:
same thing really, comment could say `// where to put payload` or `// where to put ping/pong payload`